docs/custom-features/locked-migrate.md

```diff
@@ -0,0 +1,14 @@
+# Locked Migrate
```

Comment:
Can you update the links in https://github.com/ONSdigital/dis-wagtail/blob/main/docs/custom-features/README.md and https://github.com/ONSdigital/dis-wagtail/blob/main/docs/README.md

Reply:
Done in https://github.com/ONSdigital/dis-wagtail/pull/312/commits/7f589a27bb6765ebcb3529af5d6995afba953b5e